### DISCUSSION TITLE
I've been getting this error while trying to cast a SimpleStorage contract

### DISCUSSION THREAD

**Discussion by ghost** [Cyfrin/foundry-full-course-cu#1851] [Q&A]
Failed to get resolver from the ENS registry: buffer overrun while deserializing

**Comment by newbiepersonwebthree** [comment]
hey 
I have also started the same course am am also from India  
Thought about connecting with you 
As it's good to know people with same course 
my x account is lovedeep3524 
Let's connect

**Comment by EngrPips** [comment]
Hello @Arun-Chettri, What cast command did you run before you got this error, and what were you trying to do?

**Reply by OS-Lihua** [reply]
I had the same problem

**Reply by EngrPips** [reply]
Did you resolve it already?

**Reply by imsimply** [reply]
Same.
@EngrPips just like OP, this is happening when Patrick telling how to run the cast send command to test interacting with the smart contract from the terminal (https://youtu.be/umepbfKp5rI?t=28712).

**Reply by EngrPips** [reply]
I haven't actually run into this error before, what chain did you run the process on @imsimply

**Reply by imsimply** [reply]
I was testing with the Anvil test environment.

**Reply by EngrPips** [reply]
was your `Anvil` chain running when you ran the command?

**Reply by imsimply** [reply]
yes.

**Reply by imsimply** [reply]
Never mind. Actual idiot here (was using the transaction hash instead of the contract address).

**Reply by EngrPips** [reply]
I am glad you figured it out. Keep crushing it.
